docs/Deployment/RaspberryPi4.md

```diff
@@ -0,0 +1,328 @@
+# Raspberry Pi Deployment
+
+This document guides you step by step on **how to run BTCPay Server on a Raspberry Pi 4**.
+
+The **Raspberry Pi 4** is currently the best low-cost single-board computer available.
+You can **use a Raspberry Pi 4 to run your BTCPay Server** at home for around $150 worth of parts, described below.
+
+Already have a Raspberry Pi 4 with the following specs?
+
+- 4GB memory
+- 1TB USB 3.0 SSD
+- 16GB or greater SD card
+
+If so, you can go straight to the [quickstart instructions](#quickstart).
+Otherwise, here's what you need …
+
+## Required Hardware
+
+### Raspberry Pi 4
+
+- [Raspberry Pi 4 with **4GB RAM**](https://www.canakit.com/raspberry-pi-4-4gb.html) (~$65)
+- [Sandisk 16GB SD Card](https://www.amazon.com/dp/B073K14CVB/) (~$10)
+
+Don’t settle for only 1GB or 2GB of RAM. The **4GB RAM** version is harder to find than the other versions, but you absolutely want that **4GB of RAM** for a few extra bucks, and it’s totally worth spending a few extra minutes searching on the Internet to find a vendor that has the 4GB RAM version in stock. You’ll also need an **SD card reader** if you don’t already have one.
+
+### Data Storage options
+
+- [Samsung SSD T7 1TB](https://www.amazon.com/dp/B0874XN4D8/) (~$100)
+- [SanDisk Ultra 3D 1TB](https://www.amazon.com/dp/B071KGRXRG/) (~$100)
+
+A 1TB SSD allows you to keep a full copy of the Bitcoin blockchain.
+You can also use BTCPay Server without a full copy of the Bitcoin blockchain by using the [pruning option](/Docker/#how-i-can-prune-my-nodes).
+
+### Power Adapter options
+
+- [Official Raspberry Pi 4 USB-C Power Adapter 5.1V/3A for US](https://shop.pimoroni.com/products/raspberry-pi-official-usb-c-power-supply-us?variant=29391144648787) ($10)
+- [Official Raspberry Pi 4 USB-C Power Adapter 5.1V/3A for EU](https://shop.pimoroni.com/products/raspberry-pi-official-usb-c-power-supply-eu?variant=29391130624083) ($10)
+- [Official Raspberry Pi 4 USB-C Power Adapter 5.1V/3A for AU](https://shop.pimoroni.com/products/raspberry-pi-official-usb-c-power-supply-au?variant=29391160737875) ($10)
+
+Don’t waste your time with random Chinese power adapters from Amazon, or expect that the existing ones you have at home are going to work fine. The Raspberry Pi 4 has issues with unofficial adapters, and for only $10 it’s better to just **get an official adapter** instead of learning this the hard way.
```

Comment:
```suggestion
Don’t waste your time with random cheap power adapters from Amazon, or expect that the existing ones you have at home are going to work fine. The Raspberry Pi 4 has issues with unofficial adapters, and for only $10 it’s better to just **get an official adapter** instead of learning this the hard way.
```

Reply:
Thanks!